src/libs/core/src/compiler.cpp

```diff
@@ -6454,7 +6482,11 @@ void COMPILER::SaveVariable(DATA *pV, bool bdim)
         {
             pA = TraceARoot(pV->AttributesClass, pString);
         }
+#ifdef _WIN32 // __except
```

Comment:
put some TODO tag here like FIX_LINUX
this needs to be fixed eventually because it'll lead to crashes otherwise...

Reply:
Done.